src/http/server.ts

```diff
@@ -758,32 +758,35 @@ export class Server {
   protected getResourceClass(
     request: Drash.Http.Request,
   ): Drash.Interfaces.Resource | undefined {
-    for (const resourceName in this.resources) {
-      const resource = this.resources[resourceName];
-      const pathObjs = resource.paths_parsed!;
-      for (const pathObj of pathObjs) {
-        if (pathObj.og_path === "/" && request.url_path === "/") {
-          return resource;
-        }
-
-        const pathMatchesRequestPathname = request.url_path.match(
-          pathObj.regex_path,
-        );
-        if (pathMatchesRequestPathname == null) {
-          continue;
-        }
-
-        pathMatchesRequestPathname.shift();
-        const pathParamsInKvpForm: { [key: string]: string } = {};
-        pathObj.params.forEach((paramName: string, index: number) => {
-          pathParamsInKvpForm[paramName] = pathMatchesRequestPathname[index];
-        });
+    let resource = undefined;
+    for (const regex in this.paths) {
+      if (regex === "/" && request.url_path === "/") {
+        resource = this.paths[regex];
+        break;
+      }
 
-        request.path_params = pathParamsInKvpForm;
-        return resource;
+      const pathMatchesRequestPathname = request.url_path.match(
+        regex,
```

Comment:
nitpick: do we have a linter rules for this? why is this singular arg on a new line? imo doesn't make it more readable. same with line 779, seems like we should have a bit of consistency here esp because 780 has 2 args on the same line?

Reply:
the formatting is done by the `deno fmt` command. that command is also in the CI process

Reply:
i should also mention that the CI fails if your code doesn't pass the `deno fmt` command